README.md

```diff
@@ -136,7 +136,7 @@ if (something) {
 }
 ```
 
-They work as the name implies: _if_ `something` is _truthy_ (so the boolean `true` or anything other than the empty string (`''`), `0`, `null`, or `undefined`), the code in between the curly braces runs; if not, the code between the curly braces is skipped.
+They work as the name implies: _if_ `something` is _truthy_ (so the boolean `true` or anything other than the empty string (`''`), `0`, `false`, `null`, or `undefined`), the code in between the curly braces runs; if not, the code between the curly braces is skipped.
```

Comment:
I didn't add those previous "string together" changes.. only the other ones here about adding 'false'... how do I delete that other change from the PR?